src/callbacks.c

```diff
@@ -1469,22 +1470,29 @@ void on_context_action1_activate(GtkMenuItem *menuitem, gpointer user_data)
 		!EMPTY(doc->file_type->context_action_cmd))
 	{
 		command = g_strdup(doc->file_type->context_action_cmd);
+		format = _("Cannot execute context action command \"%s\": %s. "
+			"Check the path setting in Preferences.");
 	}
 	else
 	{
 		command = g_strdup(tool_prefs.context_action_cmd);
+		format = _("Cannot execute context action command \"%s\": %s. "
+			"Check the path setting in Filetype configuration.");
 	}
 
 	/* substitute the wildcard %s and run the command if it is non empty */
 	if (G_LIKELY(!EMPTY(command)))
 	{
-		utils_str_replace_all(&command, "%s", word);
+		gchar *command_line = g_strdup(command);
 
-		if (!spawn_async(NULL, command, NULL, NULL, NULL, &error))
+		utils_str_replace_all(&command_line, "%s", word);
+
+		if (!spawn_async(NULL, command_line, NULL, NULL, NULL, &error))
 		{
-			ui_set_statusbar(TRUE, "Context action command failed: %s", error->message);
+			ui_set_statusbar(TRUE, format, command, error->message);
```

Comment:
As you can see, the two formats only differ by "the path setting in <Preferences | Filetype configuration>", so I could have used a common
_("Cannot execute context action command \"%s\": %s. "
"Check the path setting in %s.")
but wasn't sure it'll be good for the translations. If you say it's OK, I'll simplify the whole thing.

As of whether "we really want the literal %s", I'm not sure what you are talking about... We want the original command, without the selected text, which may be very long, and is an argument anyway.